src/test/java/com/ys/cat_picture/cat_image/repository/CatImageRepositoryTest.java

```diff
@@ -0,0 +1,142 @@
+package com.ys.cat_picture.cat_image.repository;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.List;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.jdbc.Sql;
+
+import com.ys.cat_picture.cat_breed.domain.CatBreed;
+import com.ys.cat_picture.cat_breed.domain.Weight;
+import com.ys.cat_picture.cat_image.domain.CatImage;
+import com.ys.cat_picture.support.RepositoryTest;
+@Sql(scripts = {"/sql/breed_sample.sql", "/sql/cat_image_sample.sql"})
+@RepositoryTest
+class CatImageRepositoryTest {
+
+	@Autowired
+	private CatImageRepository catImageRepository;
+
+	@DisplayName("이미 존재하는 데이터는 저장하지 않는다.")
+	@Test
+	void saveAllIgnore_existsDataNotInsert() {
+	    //given
+		List<CatImage> catImages = List.of(
+			new CatImage("36v", "https://cdn2.thecatapi.com/images/36v.jpg", 540, 720,
+				existsBreeds.get(0)),
+			new CatImage("24u", "https://cdn2.thecatapi.com/images/24u.png", 564, 400,
+				existsBreeds.get(1)));
+		int exitingCatImageDataSize = catImageRepository.findAll().size();
+
+		//when
+		catImageRepository.saveAllIgnore(catImages);
+
+		//then
+		assertThat(catImageRepository.findAll()).hasSize(exitingCatImageDataSize);
```

Comment:
```suggestion
		//then
		actualDataSize = catImageRepository.findAll().size();
		assertThat(actualDataSize).isEqualTo(exitingCatImageDataSize);
```

actual 로 인자값을 넘길 때, 별도로 메서드를 호출하는 과정이 없는 편이 가독성 면에서 더 좋은 것 같더라고요!